src/features/session_trace/aggregate/index.js

```diff
@@ -29,8 +29,8 @@ export class Aggregate extends AggregateBase {
     this.everHarvested = false
     /** If the harvest module is harvesting */
     this.harvesting = false
-    /** TraceStorage is the mechanism that holds, normalizes and aggregates ST nodes.  It will be accessed and purged when harvests occur */
-    this.events = new TraceStorage(this)
+    /** TraceStorage is a middleware that decides how to format data before passing events to `this.events` */
+    this.traceStorage = new TraceStorage(this)
```

Comment:
I'm not sure why you didn't run into issues but `events` is the common name for all feature data buffers, so this shouldn't be changed.

The base AggregateBase helpers references .events.